AirCasting/Dashboard/EmptyDashboardView.swift

```diff
@@ -10,13 +10,19 @@ import SwiftUI
 
 struct EmptyDashboardView: View {
     @EnvironmentObject private var tabSelection: TabBarSelection
+    @EnvironmentObject var selectedSection: SelectSection
+    var settings: BaseURLProvider
     var body: some View {
         emptyState
     }
 
     private var emptyState: some View {
         VStack(spacing: 45) {
             VStack {
+                if (selectedSection.selectedSection == .mobileDormant || selectedSection.selectedSection == .fixed) && settings.sessionSynced == false {
```

Comment:
this is a complicated boolean expression. Let's put it in a helper variable with descriptive name so we don't need to decipher this each time:
```Swift
let someThing = ...
if someThing {
    ....
}
```

Reply:
I was not able to make it into var because of ` @EnvironmentObject var selectedSection: SelectSection` but I put it into function for now at least